presto-yarn-test/etc/docker/hdp2.3/docker-compose.yml

```diff
@@ -2,31 +2,31 @@ version: '2'
 services:
   hadoop-master:
     hostname: hadoop-master
-    image: 'teradatalabs/presto-yarn-hdp2.3-master:4'
+    image: 'prestodev/presto-yarn-hdp2.3-master:30'
     privileged: true
 
   hadoop-slave1:
     hostname: 'hadoop-slave1'
-    image: 'teradatalabs/presto-yarn-hdp2.3-slave:4'
+    image: 'prestodev/presto-yarn-hdp2.3-slave:30'
```

Comment:
There is only hdp2.5, see https://github.com/Teradata/docker-images/tree/master/teradatalabs/presto-yarn-hdp2.5-slave